modules/sdk-coin-world/test/unit/transactionBuilder/send.ts

```diff
@@ -0,0 +1,16 @@
+import { getBuilder } from '../getBuilder';
+import should from 'should';
+
+describe('World Transfer Builder', () => {
+  describe('Build from TxHex', function () {
+    it('Should successfully build from txHex', async function () {
+      const txBuilder = getBuilder('tworld');
+      const txHex =
+        '0xf86902830f433a82520894e59f1cea4e0fef511e3d0f4eec44adf19c4cbeec85174876e800808225a6a0f98d41bcab494c6385cc46d21ba735ca5ec4b91f7fe44b954f7cca17f5f656f7a0125c85540c4ee242abbb29feae5d8c288ce5703c626146d68d44c7b676a56483';
+      txBuilder.from(txHex);
+      const parsedTx = await txBuilder.build();
+
+      should.exist(parsedTx.toJson());
+    });
+  });
+});
```

Comment:
Please add test for transaction building and signing. also I think for tests ethlike coin can be used. We already have tests for 2 coins [there](https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-coin-ethlike/test/unit/ethlikeCoin.ts)